sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/objects/objects.scala

```diff
@@ -1837,8 +1837,6 @@ case class GetArrayFromMap private(
     arrayGetter: MapData => ArrayData,
     elementTypeGetter: MapType => DataType) extends UnaryExpression with NonSQLExpression {
 
-  private lazy val encodedFunctionName: String = TermName(functionName).encodedName.toString
```

Comment:
this is to address https://github.com/apache/spark/pull/22745#discussion_r227407344